### DISCUSSION TITLE
tl language support for Zed

### DISCUSSION THREAD

**Discussion by rayanamal** [teal-language/tl#784] [General]
First, thank y'all for Teal. It gives me a good amount of peace of mind when writing production code in Lua.

Zed Editor [recently](https://zed.dev/blog/zed-on-linux) got Linux support. It's awesome. The only thing that keeps me on VS Code is the Teal extension, it'd be very nice to have Zed support for Teal!

**Comment by Andre-LA** [comment]
According to Zed [documentation](https://zed.dev/docs/extensions/languages), it uses Tree Sitter for syntax highlighting just as neovim and Helix.

There is a three sitter grammar for [Teal already](https://github.com/euclidianAce/tree-sitter-teal)

I did added some Helix support and shared on [codeberg](https://codeberg.org/Andre-LA/random-gists/src/branch/main/helix/extra-lang-support/teal), it's not that complete and I would like to revisit this later, but might be an start.

**Comment by hishamhm** [comment]
Thanks for the feedback! I'll move this to Discussions since it's not a bug with the compiler itself, so not something to be addressed in this repo.
